scuba/config.py

```diff
@@ -315,6 +387,11 @@ def process_command(self, command, image=None, shell=None):
                 if alias.as_root:
                     result.as_root = True
 
+                if isinstance(alias.docker_args, OverrideMixin) or result.docker_args is None:
+                    result.docker_args = alias.docker_args
+                elif alias.docker_args is not None:
+                    result.docker_args.extend(alias.docker_args)
+
                 # Merge/override the environment
                 if alias.environment:
                     result.environment.update(alias.environment)
```

Comment:
Sorry for the scope creep, but should we handle `!override` here, also? It seems like a natural application for it, considering that the existing behavior is to extend. Maybe we handle this in a subsequent MR?

Reply:
I was thinking about this also. I think it makes the most sense to create an issue and bump it to future work.

Reply:
Created issue #178.